### PR TITLE
Dev custom params enforcer tel

### PR DESCRIPTION
**Pull request by alexbpx**

### CODE REVIEW

**Comment by alexbpx** [comment]
Fixed the issues

File: perimeterx/px_config.py [hunk 1]
```diff
@@ -178,26 +182,13 @@ def __set_handler(self, function_name, config_dict):
...
-            'CUSTOM_LOGO': self.custom_logo,
-            'LOGO_VISIBILITY': self.logo_visibility,
-            'SENSITIVE_ROUTES': self.sensitive_routes,
-            'MODULE_MODE': self.module_mode,
-        }
+        config = self.__dict__
+        mutated_config = {}
+        for key, value in config.iteritems():
+            mutated_config[key[1:]] = value
+        config_dict = json.dumps(mutated_config)
```
Comment (yaronpx) [inline perimeterx/px_config.py]: you should probably return `config_dict`

Comment (alexbpx) [reply perimeterx/px_config.py]: resolved

File: perimeterx/px_config.py [hunk 2]
```diff
@@ -141,14 +146,52 @@ def sensitive_routes(self):
...
         return config_dict.get(function_name) if config_dict.get(function_name) and callable(
             config_dict.get(function_name)) else None
+
+    def __create_telemetry_config(self):
+        config = self.__dict__
+        mutated_config = {}
+        for key, value in config.iteritems():
+            mutated_config[key[1:].upper()] = value
+        return mutated_config
+
```
Comment (yaronpx) [inline perimeterx/px_config.py]: so many spaces

File: perimeterx/px_httpc.py [hunk 3]
```diff
@@ -1,51 +1,22 @@
...
             return False
 
-        logger.debug('Server call took ' + str(time.time() - start) + 'ms')
+        logger.debug('PerimeterX server call took ' + str(time.time() - start) + 'ms')
         return response
-
-    except httplib.HTTPException:
-        init(config)
+    except requests.exceptions.RequestException as e:
+        print e
```
Comment (yaronpx) [inline perimeterx/px_httpc.py]: don't `print` the exception. log it with debug level

**Pull request merged**